src/Component.php

```diff
@@ -80,11 +80,6 @@ public function getCasts()
         return $this->casts;
     }
 
-    public function render()
-    {
-        return view("livewire.{$this->getName()}");
-    }
-
```

Comment:
Just keep this as it is

Reply:
@nuernbergerA if we kept render() as it is . the generated components must implement it and can't inject dependencies in it

Reply:
I just mean to leave it here as default
I've pulled your pr and change the code like I suggested and had no errors

Reply:
if we left the render() method in the parent this exception will happen if someone tried to inject dependency

```php
 class ExecuteQuery extends Component
{
    public function render(User $user)
    {
        dd($user);
    }
}
```


![Screenshot from 2020-04-30 11-25-14](https://user-images.githubusercontent.com/17250137/80694880-65def880-8ad5-11ea-8e12-998290051aa8.png)

Reply:
Yea you're right due to PHP limitations you would have to make any additional parameter optional
```php
class ExecuteQuery extends Component
{
    public function render(User $user = null)
    {
        dd($user);
    }
}
```